packages/lexical-devtools-core/src/generateContent.ts

```diff
@@ -245,9 +249,17 @@ function normalize(text: string, obfuscateText: boolean = false) {
   return textToPrint;
 }
 
-// TODO Pass via props to allow customizability
-function printNode(node: LexicalNode, obfuscateText: boolean = false) {
-  if ($isTextNode(node)) {
+function printNode(
+  node: LexicalNode,
+  customPrintNode?: CustomPrintNode,
+  obfuscateText: boolean = false,
+) {
+  const customPrint = customPrintNode
+    ? customPrintNode(node, obfuscateText)
+    : undefined;
+  if (customPrint) {
+    return customPrint;
+  } else if ($isTextNode(node)) {
```

Comment:
since the if block above returns we dont need to wrap this in an else

```suggestion
 if ($isTextNode(node)) {
```

Reply:
I was just following the existing pattern as all the following `else if` blocks all return. Strictly they could all be changed to `if` blocks but I think that would make the code harder to read. I'm happy to change them all to `if` blocks if you prefer that?

Reply:
the custom print case is a seperate case from the default print so its clearer to have it in its own if block rather then lump it tgt with the default print conditions

Reply:
That makes sense also, done.